libs/sdk-core/src/models.rs

```diff
@@ -682,6 +685,18 @@ pub struct ReverseSwapFeesRequest {
     pub send_amount_sat: Option<u64>,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct MaxReverseSwapAmountResponse {
+    pub total_sat: u64,
+}
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct MaxChannelAmount {
+    pub channel_id: String,
+    pub amount_msat: u64,
+    pub path: PaymentPath,
+}
+
```

Comment:
Can we add documentation for these newly introduced structs and their fields for the end-users?
They don't need to be in-depth, a brief explanation is enough.

Reply:
Done